swift/ios-application/build.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    dependencies {
+        classpath "org.gradle.samples:ios-plugin:latest.integration"
+    }
+}
+
+plugins {
+    id 'swift-application'
+    id 'xcode'
+}
+
+apply plugin: 'org.gradle.samples.ios-application'
+
+group = "org.gradle.samples"
+
+// Add iOS specific resource to Xcode IDE project
+file('src/main/resources/Base.lproj').listFiles().each {
+    xcode.project.groups.sources.from it
+}
+xcode.project.groups.sources.from 'src/main/resources/Info.plist'
```

Comment:
`xcode.project.groups` is an internal API (and won't work from Kotlin, for example).

Reply:
Good point, this was moved inside the plugin. I guess these internal API will have to be opened up at some point. It would be preferable to align them with Idea, Eclipse and Visual Studio.

Reply:
We should do that now, or remove this from the sample. Samples need to reflect how we suggest people solve the problem, and I don't think we should ever suggest using internal APIs.

Reply:
I have open issue https://github.com/gradle/gradle-native/issues/654 to track this discussion.

Reply:
I also removed the internal API usage in the sample: https://github.com/gradle/native-samples/commit/5fc4478e1445f7d16c34b8cc8621f4625a9d2f62